lib/web/ui/resetpasswordtoken.go

```diff
@@ -28,7 +28,6 @@ type ResetPasswordToken struct {
 	QRCode []byte `json:"qrCode,omitempty"`
 	// Expiry is token expiration time
 	Expiry time.Time `json:"expiry,omitempty"`
-	// IsResetSecondFactor is a flag that when true, token is used
-	// to reset secondfactor, else default is reset password.
-	IsResetSecondFactor bool `json:"isResetSecondFactor,omitempty"`
+	// RecoveryType is the recover type user requested upon starting recovery.
+	RecoveryType string `json:"recoverType,omitempty"`
```

Comment:
In the request for starting the recovery process, it expects `recoveryType`:
https://github.com/gravitational/teleport.e/blob/eeb41ce45f257f10f189048868fe5a01fcab7291/lib/web/accountrecovery.go#L54-L55

But in this response it returns `recoverType`, perhaps change this to `recoveryType` to keep it consistent?

Reply:
good catch, revised